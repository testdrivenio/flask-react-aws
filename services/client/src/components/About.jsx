import React from 'react';

const About = () => (
  <div>
    <h1 className="title is-1">About</h1>
    <hr /><br />
    <p className="content">Add something relevant here.</p>
  </div>
)

export default About;
