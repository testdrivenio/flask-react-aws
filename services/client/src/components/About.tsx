import React from "react";
import { Box, Heading, Text, Divider, VStack } from "@chakra-ui/react";

const About = () => (
  <Box p={4} maxW="1200px" mx="auto" mt={12}>
    <VStack spacing={6} align="stretch">
      <Heading as="h1" size="2xl">
        About
      </Heading>
      <Divider />
      <Text fontSize="lg" className="content">
        Add something relevant here.
      </Text>
    </VStack>
  </Box>
);

export default About;
