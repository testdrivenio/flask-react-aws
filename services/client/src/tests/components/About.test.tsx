import React from "react";
import { render, cleanup } from "../test-utils";
import { it, expect, afterEach } from "vitest";
import "@testing-library/jest-dom/vitest";

import About from "../../components/About";

afterEach(cleanup);

it("renders properly", () => {
  const { getByText } = render(<About />);
  expect(getByText("Add something relevant here.")).toHaveClass("content");
});

it("renders", () => {
  const { asFragment } = render(<About />);
  expect(asFragment()).toMatchSnapshot();
});
