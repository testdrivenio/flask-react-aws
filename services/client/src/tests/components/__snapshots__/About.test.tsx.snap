// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders 1`] = `
<DocumentFragment>
  <div
    class="css-zs4oec"
  >
    <div
      class="chakra-stack css-42ki38"
    >
      <h1
        class="chakra-heading css-1jkx6l7"
      >
        About
      </h1>
      <hr
        aria-orientation="horizontal"
        class="chakra-divider css-svjswr"
      />
      <p
        class="chakra-text content css-i3jkqk"
      >
        Add something relevant here.
      </p>
    </div>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</DocumentFragment>
`;
